src/frame-writer.cpp

```diff
@@ -250,10 +289,18 @@ void FrameWriter::init_audio_stream()
     }
 
     audioCodecCtx = audioStream->codec;
-    audioCodecCtx->bit_rate = lrintf(128000.0f);
-    audioCodecCtx->sample_fmt = get_codec_sample_fmt(codec);
+    if (params.codec == "aac"){
```

Comment:
I looked into it again, and it seems that we do not need to set the bit rate at all. You can delete this if altogether.